docs/debian12-install-guide.md

```diff
@@ -147,7 +147,7 @@ make -j $(nproc) install
 ```bash
 cp -n ~/server/etc/authserver.conf.dist ~/server/etc/authserver.conf
 cp -n ~/server/etc/worldserver.conf.dist ~/server/etc/worldserver.conf
-sudo sed -i -E 's/DataDir = "."/DataDir = "\~\/server\/data"/' ~/server/etc/worldserver.conf
+sudo sed -i -E 's|^DataDir = *$|DataDir = "../data"|' ~/server/etc/worldserver.conf
```

Comment:
`../data` depends on the current directory of the terminal `"\~\/server\/data"/'` does not, why did you change it?

Reply:
Setting Datadir as ~/server/data was causing the mmaps to not load properly. (showed mmap:0 in .gps)

Reply:
To add to this, `DataDir = "../data"` was working for me, but was not working for someone else also on Debian 12.  Absolute path works in both of our cases so that seems the way to go.